lib/moonshot/cli.rb

```diff
@@ -106,7 +128,11 @@ def create
     end
 
     desc :update, 'Update the CloudFormation stack within an environment.'
-    option :show_all_events, desc: 'Show all stack events during update. (Default: errors only)'
+    option :parameter_strategy, desc: 'Override default parameter strategy.'
+    option(
+      :show_all_events,
+      type: :string,
+      desc: 'Show all stack events during update. (Default: errors only)')
```

Comment:
No, thats a mistake, I wanted to specify the type for `parameter_strategy`, not `show_all_events`. Shall I also modify `show_all_events `'s type to boolean?